src/request.ts

```diff
@@ -1186,6 +1231,46 @@ class DatastoreRequest {
     this.save(entities, callback);
   }
 
+  merge(entities: Entities): Promise<CommitResponse>;
```

Comment:
Does this new method belong to this PR?

Reply:
oops! my bad this is not involve in this PR!